content/04.discussion.md

```diff
@@ -27,6 +27,8 @@ used efficiently and effectively for compositional analysis of shotgun
 metagenome data sets with k-mers. In particular, FracMinHash is
 competitive in accuracy with extant MinHash-based techniques for
 containment analysis, while also supporting Jaccard similarity.
+Further analytical results for FracMinHash are available in
```

Comment:
In addition, FracMinHash can be used to obtain point estimates of and confidence intervals around mutation rates and evolutionary distances, see [@dk_fracminhash] for these and other analytical results.

Reply:
updated in 5603929